frontend/src/components/MapView/DateSelector/utils.ts

```diff
@@ -151,5 +151,5 @@ export function findDateIndex(
 }
 
 export function formatDate(date: number): string {
-  return moment(date + USER_DATE_OFFSET).format(MONTH_FIRST_DATE_FORMAT);
+  return moment(date).format(MONTH_FIRST_DATE_FORMAT);
```

Comment:
@HarryMytilinaios could you do a bit more QA?
This is the only thing that actually changed in this PR. Other changes are just cleanups.

Reply:
I think this is good to go. I just did a little bit more QA on this and it seems that it works as expected. I added multiple layers and changed my timezone and it seems pretty solid from my side.